src/service/transaction.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+import "blockchain_routing_address.proto";
+import "blockchain_txn.proto";
+
+enum txn_status {
+  pending = 0;
+  cleared = 1;
```

Comment:
We wont need cleared here anymore as the txn mgr wont be hold this in the revised version of the backend changes